common/common.scss

```diff
@@ -212,3 +212,37 @@ input {
 .alert-emails-disabled {
     display: none;
 }
+
+.badge-card {
```

Comment:
To keep `common.scss` simple and tidy, I would prefer splitting badge related stylings to a new scss file in `common/badgeCard.scss`. Then you may import the new scss for the same stylings.

```scss
// In common.scss
import `./badgeCard`;
```

```scss
// common/badgeCard.scss
.badge-card {
  // Rest code..
}
```

Reply:
Currently there's only one file, is it necessary to split into separate files, when there's not yet a lot of code?

Reply:
For the existing code, we can refactor it gradually. However, for the newly added code, if the effort is nearly the same, I think we can consider more about code's maintainability and readability.

The `common.scss` is already more than 200 lines of code, and it's not as simple as it should be. Besides, splitting the badge stylings into an individual file helps to save time spent in locating styles of badge cards since you could open `common/badgeCard.scss` instantly for your future modification, instead of searching thru the whole `common.scss`.

> The ratio of time spent reading versus writing is well over 10 to 1 - Robert Martin 

Therefore, I encourage that we'd better make every effort to write the code as clean as possible.

Reply:
Done.